pflake8 6.0.0 overrides some pyproject.toml parameters with default values when jobs>1

Each worker of flake8 ignores `pyproject.toml` in `--jobs` > 1 executions.

Steps to reproduce:

I have to files (`file1.py`, `file2.py`) with same content:
```
print("long line =============================================================================================")
```
It violates 2 rules: W292 (newline in EOF) and E501 (max-line-length)

In my `pyproject.toml` file I have following flake8 configuration:
```
[tool.flake8]
max-line-length = 120
ignore = "W292"
```
Running `pflake8 -vv --jobs=2` on version 5.0.4. works fine. However same command on version 6.0.0 raises E501 with default 79 char parameter.
```
.\file1.py:1:80: E501 line too long (112 > 79 characters)
.\file2.py:1:80: E501 line too long (112 > 79 characters)
```

Full execution logs are attached:
[5_0_4.txt](https://github.com/csachs/pyproject-flake8/files/10217759/5_0_4.txt)
[6_0_0.txt](https://github.com/csachs/pyproject-flake8/files/10217761/6_0_0.txt)

On line 92 of 6_0_0.txt I can see that parameter from toml file is used. But then each worker reads config again but this time not reading toml file configuration.

--- a/pflake8/__init__.py
+++ b/pflake8/__init__.py
@@ -4,9 +4,10 @@ __version__ = '6.0.0'
 
 import ast
 import configparser
+import multiprocessing.pool
 import sys
-from types import ModuleType
 from pathlib import Path
+from types import ModuleType
 
 import flake8.main.cli
 import flake8.options.config
@@ -103,6 +104,35 @@ class FixFilenames(ast.NodeTransformer):
 
 FixFilenames.apply()
 
+
+def _pool_init(real, *args, **kwargs):
+    # by having the multiprocessing Pool load this function
+    # the monkeypatches are loaded as well
+    return real(*args, **kwargs)
+
+
+class PatchingPool(multiprocessing.pool.Pool):
+    def __init__(
+        self,
+        processes=None,
+        initializer=None,
+        initargs=(),
+        maxtasksperchild=None,
+        context=None,
+        **kwargs
+    ):
+        super().__init__(
+            processes=processes,
+            initializer=_pool_init,
+            initargs=(initializer,) + initargs,
+            maxtasksperchild=maxtasksperchild,
+            context=context,
+            **kwargs
+        )
+
+
+if multiprocessing.get_start_method() == "spawn":
+    multiprocessing.pool.Pool = PatchingPool
 
 main = flake8.main.cli.main
 

